cobalt/site/docs/reference/starboard/modules/14/memory.md

```diff
@@ -30,8 +30,9 @@ and SbMemoryDeallocate rather than SbMemoryAllocateUnchecked and SbMemoryFree.
 
 ### SbMemoryMapFlags
 
-The bitwise OR of these flags should be passed to SbMemoryMap to indicate how
-the mapped memory can be used.
+TODO: Remove the definition once the memory_mapped_file.h extension is
```

Comment:
This todo in the docs looks out of place. Maybe we restore old comment and move TODO up so that Doxygen doesn't pick it up ?